projects/bp-gallery/cypress/e2e/picture.cy.ts

```diff
@@ -48,4 +48,12 @@ describe('Picture View called via link as a curator', () => {
     cy.get('.picture-sidebar .quick-access-buttons').click();
     cy.get('.picture-info-field').should('have.length', pictureInfoFields.length);
   });
+
+  it('should have date estimate functionality', () => {
+    cy.get('.date-indicator').click();
+    cy.get('.date-range-picker .MuiCheckbox-root:not(.Mui-checked)');
+    cy.get('.date-range-picker .MuiCheckbox-root').click();
+    cy.get('.date-range-picker .MuiCheckbox-root.Mui-checked');
+    cy.get('.date-range-picker .MuiCheckbox-root').click().type('{esc}');
```

Comment:
According to [this cypress guide](https://docs.cypress.io/guides/core-concepts/retry-ability#Actions-should-be-at-the-end-of-chains-not-the-middle), it's better to only put one action at the end of a chain. So please put the `type` on the next line, repeating the `get`.